.github/workflows/codeql-analysis.yml

```
@@ -42,6 +42,16 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: 17
+          distribution: 'temurin'
+          server-id: jfrog-central
+          server-username: INTERNAL_USERNAME
+          server-password: INTERNAL_PASSWORD
```

We may omit server properties, right @zcgandcomp?

```suggestion
          distribution: 'temurin'
```